src/data/__tests__/calculate.test.ts

```diff
@@ -135,36 +135,38 @@ describe('calculate', () => {
     const response = calcValue(data)
     // average * 2 people * outdoor * 1 hr * their mask * your mask
     expect(response).toBeCloseTo(
-      ((((PREVALENCE * 2) / 20) * 0.06 * B117_CONTAGIOUSNESS_ADJUSTMENT) /
-        4 /
-        1) *
+      ((PREVALENCE * 2) / 20) *
+        0.06 *
+        B117_CONTAGIOUSNESS_ADJUSTMENT *
+        0.3333 *
+        0.6667 *
         1e6,
     )
   })
 
   it.each`
     scenario                                                          | result
-    ${'Outdoor masked hangout with 2 other people'}                   | ${9 * B117_CONTAGIOUSNESS_ADJUSTMENT}
+    ${'Outdoor masked hangout with 2 other people'}                   | ${8 * B117_CONTAGIOUSNESS_ADJUSTMENT}
     ${'Indoor unmasked hangout with 2 other people'}                  | ${720 * B117_CONTAGIOUSNESS_ADJUSTMENT}
     ${'Car ride with 1 other person for 15 mins'}                     | ${90 * B117_CONTAGIOUSNESS_ADJUSTMENT}
     ${'One-night stand with a random person'}                         | ${2880}
-    ${'Live-in partner who has no indoor interactions besides you'}   | ${21.6 * B117_CONTAGIOUSNESS_ADJUSTMENT}
-    ${'Grocery store for 60 minutes (average # of shoppers)'}         | ${90 * B117_CONTAGIOUSNESS_ADJUSTMENT}
-    ${'Grocery store for 60 minutes (few other shoppers)'}            | ${45 * B117_CONTAGIOUSNESS_ADJUSTMENT}
-    ${'Plane ride (full flight)'}                                     | ${369 * B117_CONTAGIOUSNESS_ADJUSTMENT}
-    ${'Plane ride (middle seat empty)'}                               | ${180 * B117_CONTAGIOUSNESS_ADJUSTMENT}
+    ${'Live-in partner who has no indoor interactions besides you'}   | ${29 * B117_CONTAGIOUSNESS_ADJUSTMENT}
```

Comment:
This increased, but it should have stayed the same - live-in partner uses a surgical mask which was not modified by the data change. If this test is passing, something seems off to me...

Reply:
Nope, it checks out - the living alone multiplier is slightly higher because it assumes the user wears a surgical mask but others wear a cloth mask.

I still, months later, think it should all be cloth mask, but it doesn't really matter.